tests/assistant_spec.js

```diff
@@ -16,50 +16,45 @@ const {MATH_RULES_LOCALES} = require('../lib/math-translator');
  */
 function makeGraphie() {
     const baseURL = 'web+graphie://ka-perseus-graphie.s3.amazonaws.com';
-    const id = Date.now() + (1000 * Math.random() | 0);
+    const id = Date.now() + (100000 * Math.random() | 0);
+    console.log(id);
```

Comment:
Can this console.log be removed?

Reply:
Huh, I was too hasty. It's weird that ESLint didn't catch it. Apparently, the new version does not have the `no-console` rule by default, so I added it to `.eslintrc`. Thanks!